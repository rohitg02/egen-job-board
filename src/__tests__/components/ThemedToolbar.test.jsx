import React from 'react';
import { shallow } from '../../enzyme';

import ThemedToolbar from '../../components/ThemedToolbar';

describe('ThemedToolbar component', () => {

    it('should render ThemedToolbar component', () => {
      const wrapper = shallow(<ThemedToolbar />);
      expect(wrapper).toMatchSnapshot();
    });
});
