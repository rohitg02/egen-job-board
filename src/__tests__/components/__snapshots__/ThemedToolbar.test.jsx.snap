// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemedToolbar component should render ThemedToolbar component 1`] = `ShallowWrapper {}`;
